### PR TITLE
only show the first 25 releases in "Releases Table"

### PR DESCRIPTION
**Pull request by Bjwebb**

### CODE REVIEW

**Comment by Bjwebb** [comment]
[Quick fixes for large files](https://trello.com/c/pZJldiNG/70-quick-fixes-for-large-files)

File: cove_ocds/templates/cove_ocds/explore_release.html [hunk 1]
```diff
@@ -341,6 +341,11 @@ <h4 class="panel-title">
         </h4>
       </div>
       <div class="panel-body">
+        {% if releases|length > 25 %}
+        <p>
+          {%blocktrans%}Only showing the first 25 releases. If you want to see more, look at the spreadsheet in "Convert" above.{%endblocktrans%}
```
Comment (robredpath) [inline cove_ocds/templates/cove_ocds/explore_release.html:346]: @Bjwebb don't we normally put spaces in `{% template commands %}`?

As for copy, I think:

"Showing the first 25 releases. To explore all your data in a tabular format, convert it to a spreadsheet using the "Convert" section, above".  

@jpmckinney , how does that sound to you?

Comment (jpmckinney) [reply cove_ocds/templates/cove_ocds/explore_release.html:346]: @robredpath Sounds good!

Comment (Bjwebb) [reply cove_ocds/templates/cove_ocds/explore_release.html:346]: > @Bjwebb don't we normally put spaces in {% template commands %}?

Normally. Altough I copied this from further down the file. I'll update this along with the text.

**Comment by Bjwebb** [comment]
This is what the new text looks like:
Screenshot from 2020-05-06 10-48-39

Here's the change in file size of the request for some different files:

File|Valid?|File Size|Response size before PR|Response size after PR
---|---|---|---|---
[releases.ocds.json](https://static.data.gouv.fr/resources/fichiers-consolides-des-donnees-essentielles-de-la-commande-publique/20200410-172633/releases.ocds.json)|Valid|366MB|156MB|3.92 MB
repeated_errors_repeated.json (not public data)|Invalid|359MB|65.8 MB|27.32 MB
[badfile_repeated.json](http://temp.dev3.default.opendataservices.uk0.bigv.io/ocds_sample_data/badfile_repeated.json) ([script to generate](https://gist.github.com/Bjwebb/2507c57e5ce9fea9a833135d2142a671))|Invalid|341MB|174.18 MB|150 MB

This change makes a big difference for valid data, but not so much for invalid data (which has long lists of information about each error).

**Comment by jpmckinney** [comment]
@Bjwebb Where is repeated_errors_repeated.json stored? Can we have a private repo with useful test data, if some if it can't be made public?

**Comment by Bjwebb** [comment]
@jpmckinney Currently its just on my desktop.

I can't put it in a GitHub repository directly, because there's a limit on file size of 100 MB. It's a file I've created by programmatically by repeating some smaller data, so I could make a repository with a script to recreate it.

**Comment by jpmckinney** [comment]
Can we make the number 25 a configurable setting? Feel free to create a new issue if you prefer to do that in a second PR.

**Comment by Bjwebb** [comment]
Private repo with sample data exists now: https://github.com/open-contracting/sample-data-private

**Pull request merged**